### DISCUSSION TITLE
Troubleshooting with Polyglot

### DISCUSSION THREAD

**Discussion by Selinady** [mclemente/fvtt-module-polyglot#322] [Q&A]
Hello everyone,
my problem is a bit odd and I'm getting really stressed by it. I was trying to fix my Polyglot, because last time I played with my peoples, they weren't able to send messages in other languages. Long story short. I'm not able to make this work for my peoples. The drop down menu for Scramble OOC Chat Messages is empty. Something is off, please help me...

Here some screenshots...

![grafik](https://github.com/mclemente/fvtt-module-polyglot/assets/147001114/3f9ba06a-54a3-4382-8c95-1c6d35286789)
![grafik](https://github.com/mclemente/fvtt-module-polyglot/assets/147001114/ff386897-73a1-49b0-b5d7-7cd062a79d9f)
![grafik](https://github.com/mclemente/fvtt-module-polyglot/assets/147001114/6866d841-e3a2-4bb6-844e-a94013ff90f4)
![grafik](https://github.com/mclemente/fvtt-module-polyglot/assets/147001114/225437a6-9a85-40a9-afe3-92a5e5e99008)
![grafik](https://github.com/mclemente/fvtt-module-polyglot/assets/147001114/d88b69df-5d7a-4c32-873d-d5d407a8510d)
![grafik](https://github.com/mclemente/fvtt-module-polyglot/assets/147001114/1eef85b0-2ee3-4b90-9c93-b88763b936fa)

**Comment by mclemente** [comment]
It was a misconfiguration of the menu, I'm sorry for your troubles. Update to v2.3.4 and let me know if the issue persists.

**Reply by Selinady** [reply]
It's fixed. Sorry for the trouble. You made a really awesome module, thank you for this.
